src/textacy/extract.py

```diff
@@ -162,8 +162,8 @@ def ngrams(
         )
     if min_freq > 1:
         ngrams_ = list(ngrams_)
-        freqs = itertoolz.frequencies(ngram.lower_ for ngram in ngrams_)
-        ngrams_ = (ngram for ngram in ngrams_ if freqs[ngram.lower_] >= min_freq)
+        freqs = itertoolz.frequencies(ngram.lemma_.lower() for ngram in ngrams_)
+        ngrams_ = (ngram for ngram in ngrams_ if freqs[ngram.lemma_.lower()] >= min_freq)
```

Comment:
Again, is this a spaCy v3 issue, or just a difference in expectations? Lemmatizing and lowercasing under the hood when determining ngram frequencies felt like more magic than I would expect, so I stopped at just lowercasing. I'm open to reconsidering, but would want to check that lemmas are available for all languages — I don't _think_ so, in which case this feature would fail for certain langs, but I'm not sure.

Reply:
Okay, I just saw that they removed `Span.lower_` from the v3 API, so the direct equivalent here would be to use `.text.lower()`. I'm currently working through a pile of bugs/errors on the v3-based develop branch, so I'll fix it there — just have to get everything working again before I start making larger v3-oriented changes. :)